tests/integration/version_test.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+# Ensure CHANGELOG version and merlin/__init__.py version are equal
+grep CHANGELOG.md -m 1 -e "## \[\d\+\.\d\+\.\d\+\]" > GREP1
+grep merlin/__init__.py -m 1 -e "\"\d\+\.\d\+\.\d\+\"" > GREP2
+grep -o GREP1 -e "\d\+\.\d\+\.\d\+" > GREP3
+grep -o GREP2 -e "\d\+\.\d\+\.\d\+" > GREP4
+if cmp GREP3 GREP4
```

Comment:
You can use ! to negate the test to remove the  : and else.

Reply:
Fixed.